python/sglang/srt/layers/quantization/fp8_kernel.py

```diff
@@ -59,6 +60,7 @@ def deep_gemm_fp8_fp8_bf16_nt(
         Bs: torch.Tensor,
         C: torch.Tensor,
     ) -> None:
+        warmup_deepgemm.capturer_on_kernel_executed((A, As), (B, Bs))
```

Comment:
Just curious, why add warmup_deepgemm before gemm_fp8_fp8_bf16_nt

Reply:
Because need to capture it (when user enables the capture mode)